src/griffe/finder.py

```diff
@@ -284,12 +286,25 @@ def _module_depth(name_parts_and_path: NamePartsAndPathType) -> int:
     return len(name_parts_and_path[0])
 
 
-def _handle_pth_file(path):
-    # support for .pth files pointing to a directory
-    instructions = path.read_text().strip("\n").split(";")
-    added_dir = Path(instructions[0])
-    if added_dir.exists():
-        return added_dir
+def _handle_pth_file(path) -> list[Path]:  # noqa: WPS231
+    # Support for .pth files pointing to directories.
+    # From https://docs.python.org/3/library/site.html:
+    # A path configuration file is a file whose name has the form name.pth
+    # and exists in one of the four directories mentioned above;
+    # its contents are additional items (one per line) to be added to sys.path.
+    # Non-existing items are never added to sys.path,
+    # and no check is made that the item refers to a directory rather than a file.
+    # No item is added to sys.path more than once.
+    # Blank lines and lines beginning with # are skipped.
+    # Lines starting with import (followed by space or tab) are executed.
+    directories = []
+    for line in path.read_text().strip().splitlines(keepends=False):
+        line = line.strip()
+        if line and not line.startswith("#") and not _re_import_line.search(line):
+            if os.path.exists(line):
+                directories.append(Path(line))
+    if directories:
+        return directories
     raise UnhandledPthFileError(path)
```

Comment:
Hmm, maybe we shouldn't raise at all. If no directories are found, just let it flow :thinking: ?

Reply:
Hello, thanks and sorry for the late response!
I do agree, we should probably let it flow, as users have little control over dependencies `.pth` files.

Reply:
Done.